x-pack/plugins/cases/public/components/configure_cases/connectors.tsx

```diff
@@ -113,7 +114,6 @@ const ConnectorsComponent: React.FC<Props> = ({
                   isLoading={isLoading}
                   onChange={onChangeConnector}
                   data-test-subj="case-connectors-dropdown"
-                  appendAddConnectorButton={true}
```

Comment:
nit: Now that the `appendAddConnectorButton` is not used anymore what about removing the functionality (`appendAddConnectorButton`) from the `ConnectorsDropdown` component?

Reply:
done, thanks!